dockstore-webservice/src/main/java/io/dockstore/webservice/resources/AbstractWorkflowResource.java

```diff
@@ -536,4 +558,24 @@ private String gitHubAppSetup(String installationId) {
         }
         return installationAccessToken;
     }
+
+    /**
+     * Parse git references (ex. refs/heads/feature/foobar) and returns a pair of the branch type and name
+     * @param gitReference Git Reference of the form refs/tags/name or refs/heads/name
+     * @return Pair of reference information
+     */
+    private Pair<String, String> parseGitHubReference(String gitReference) {
```

Comment:
I have been making too many inner classes recently so wanted to reduce that, but ya I'll just edit it to only return the name. I was just returning both in case in the future someone needed both.